tests/network/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 go_library(
     name = "go_default_library",
     srcs = [
+        "bridge_hotplug.go",
```

Comment:
Could you please keep `hotplug` as the prefix?
That is the main subject, the binding type is secondary.